Example/Example.csproj

```diff
@@ -5,10 +5,10 @@
     <!-- Version numbers are automatically updated by gitversion when a release is released -->
     <!-- In the source tree the version will always be 1.0 for all projects. -->
     <!-- Do not modify these. -->
-    <AssemblyVersion>1.13.6.0</AssemblyVersion>
-    <FileVersion>1.13.6.0</FileVersion>
-    <Version>1.13.6-pre0002</Version>
-    <InformationalVersion>1.13.6-pre.2+Branch.develop.Sha.90208d6577b9d34bf0c4e9057b4227297bc9118d</InformationalVersion>
+    <AssemblyVersion>1.14.0.0</AssemblyVersion>
+    <FileVersion>1.14.0.0</FileVersion>
+    <Version>1.14.0-v1-develop-branc0115</Version>
+    <InformationalVersion>1.14.0-v1-develop-branching-update.115+Branch.v1-develop-branching-update.Sha.196872f2340046682a9fc98bceba4894e866e1c5</InformationalVersion>

```

Comment:
Since this information is repeated on various projects it is possible to use some environment variable or file where the projects can read, like something `<AssemblyVersion>$(AssemblyVersion)</AssemblyVersion>`?

Reply:
Not sure. I am trying to figure out how to keep this simple.